src/components/record/RecordForm.js

```diff
@@ -45,6 +45,20 @@ export function extendSchemaWithAttachment(
   };
 }
 
+export function extendUIWithKintoFields(
+  uiSchema: Object,
+  isCreate: boolean
+): Object {
+  return {
+    ...uiSchema,
+    id: {
+      "ui:widget": "text",
+      "ui:disabled": !isCreate,
```

Comment:
Should this maybe be `ui:readonly` instead of disabled?

Reply:
Good question...

We don't need the `id` value to get passed with the form data.

Should the id field be focusable? included in the tabbed field navigation?

Reply:
I just thought a user might want to copy-paste the ID. I guess they can already grab it from the breadcrumbs, though, so maybe not a big deal?

Reply:
I would leave it disabled :)